### DISCUSSION TITLE
蜜柑计划app-1.0.0 rc-26

### DISCUSSION THREAD

**Discussion by blusedark** [iota9star/mikan_flutter#19] [General]
点开详情页底下有显示代码，应该不正常吗？
![IMG_20211230_102603](https://user-images.githubusercontent.com/66894797/147716841-9bbccbc3-c8fc-441f-9dc6-9a14ac9abb35.jpg)

**Comment by iota9star** [comment]
这个是正常的，没有问题。有的网页是有这样的问题
